openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/TopBar.vue

```diff
@@ -22,18 +22,23 @@
 
 <template>
   <mounting-portal mount-to="#openc3-menu" append>
-    <div class="v-toolbar__content">
+    <v-row no-gutters
+      ><v-col align-self="end">
+        <span class="app-title mr-2">{{ title }}</span></v-col
```

Comment:
I considered adding the version here like [astro](https://www.astrouxds.com/components/global-status-bar/) does but it takes up space and seems unnecessary since we have it in the footer